src/docs/deployment/flavors.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Creating flavors for Flutter
+description: How to create build flavors specific to different devices.
+---
+
+Do you need to set up product flavors for different devices?
```

Comment:
`Do you need to set up product flavors for different development environments or release types?` may make more sense (flavors aren't necessarily device specific).